callouts/python/extproc/example/add_body/service_callout_example.py

```diff
@@ -31,13 +32,13 @@ def on_request_body(
       self, body: service_pb2.HttpBody, context: ServicerContext
   ) -> service_pb2.BodyResponse:
     """Custom processor on the request body."""
-    return callout_server.add_body_mutation(body='-added-body')
+    return callout_tools.add_body_mutation(body='-added-body')
```

Comment:
why is this change being done?

Reply:
It looks like the swap over to callout tools was never completed for this file, this is good, there should have been a test to catch this. Thank you for adding one.